src/test/java/org/sagebionetworks/web/unitclient/widget/entity/tabs/DatasetsTabTest.java

```diff
@@ -463,49 +463,6 @@ public void testShowProjectLoadError() {
 		verify(mockSynapseAlert).handleException(projectLoadError);
 	}
 
-	@Test
-	public void testEditorCanDownloadLatestVersion() {
```

Comment:
This behavior is now in the TableEntityWidget so tests were removed here and new tests were added there.